src/actions.ml

```diff
@@ -257,6 +257,79 @@ let send_doc_url ~bot_info ~github_repo_full_name job_info =
   | _ ->
       Lwt.return_unit
 
+let send_bench_url ~bot_info ~github_repo_full_name job_info (gh_owner, gh_repo)
+    ~external_id =
+  match (github_repo_full_name, job_info.build_name) with
+  | "coq/coq", "bench" ->
+      Lwt_io.print "Pushing status check for bench job.\n"
+      <&>
+      let url = f "https://gitlab.com/coq/coq/-/jobs/%d" job_info.build_id in
+      let state = job_info.build_status in
+      let context = "bench" in
+      let description_base = " - Link to bench job" in
+      url |> Uri.of_string |> Client.get
+      >>= fun (resp, _) ->
+      let status_code = resp |> Response.status |> Code.code_of_status in
+      if Int.equal 200 status_code then
+        let state, status, conclusion, description =
+          match state with
+          | "success" ->
+              ( "success"
+              , COMPLETED
+              , Some SUCCESS
+              , "Bench completed successfully" )
+          | "created" ->
+              ( "pending"
+              , QUEUED
+              , None
+              , "Bench requested" )
```

Comment:
I'm guessing just removing this should be enough to fix this issue:
```suggestion
```

Reply:
But possibly a better option would just be to replace "created" with "pending":
```suggestion
          | "pending" ->
              ( "pending"
              , QUEUED
              , None
              , "Bench requested" )
```

Currently, the bot log contains a lot of "Unknown job status: pending"

Reply:
Another bug is that a bench hasn't even been requested. I think I will settle for not including this.

Reply:
>Another bug is that a bench hasn't even been requested

This was my point actually :wink: